### DISCUSSION TITLE
Image component doesn't show images linked from http when deployed to Vercel

### DISCUSSION THREAD

**Discussion by CzechCoder** [vercel/next.js#60979] [Help]
### Summary

Hello, in Next.js 13 I'm having problem displaying images that are linked to http, the links look like this: http:/tomasburian.com/dev/cdn/mclaren-store/01.jpg

I feed this link to src of Next's Image component and it works on dev, but when I deploy to Vercel, it starts giving error 404 and the link looks like this: https://mclaren-store-dashboard.vercel.app/_next/image?url=http%3A%2Ftomasburian.com%2Fdev%2Fcdn%2Fmclaren-store%2F01.jpg&w=48&q=75

Now, my next.config.js looks like this:

const nextConfig = {
  reactStrictMode: true,
  images: {
    remotePatterns: [
      {
        protocol: "http",
        hostname: "tomasburian.com",
        port: "",
        pathname: "/dev/**",
      },
    ],
  },
};
So I'm not sure where the problem is, any help please?

**Comment by MohsinKamalKhan** [comment]
found any solution?
